graphql/distro_resolver.go

```diff
@@ -11,9 +11,23 @@ import (
 	"time"
 
 	"github.com/evergreen-ci/birch"
+	"github.com/evergreen-ci/evergreen"
 	"github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/evergreen-ci/utility"
 )
 
+// CloneMethod is the resolver for the cloneMethod field.
+func (r *distroResolver) CloneMethod(ctx context.Context, obj *model.APIDistro) (CloneMethod, error) {
+	switch *obj.CloneMethod {
```

Comment:
Can we validate that obj is not nil before we dereference the pointer and also do `switch utility.FromStringPtr(obj.CloneMethod)`